cli/tauri.js/src/helpers/download-cli.ts

```diff
@@ -0,0 +1,39 @@
+import stream from 'stream'
+import { promisify } from 'util'
+import fs from 'fs'
+import got from 'got'
+import { CargoManifest } from '../types/cargo'
+import path from 'path'
+const pipeline = promisify(stream.pipeline)
+const currentTauriCliVersion = (): string => {
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-var-requires, @typescript-eslint/no-unsafe-member-access
+  const tauriCliManifest = require('../../../core/Cargo.toml') as CargoManifest
+  return tauriCliManifest.package.version
+}
+
+const downloadCli = async (): Promise<void> => {
+  const version = currentTauriCliVersion()
+  let platform: string = process.platform
+  if (platform === 'win32') {
+    platform = 'windows'
+  } else if (platform === 'linux') {
+    platform = 'linux'
+  } else if (platform === 'darwin') {
+    platform = 'macos'
+  } else {
+    throw Error('Unsupported platform')
+  }
+  const exe = platform === 'windows' ? '.exe' : ''
+  const url = `https://github.com/nklayman/tauri-binary-releases/releases/download/tauri-cli-v${version}/tauri-cli_${platform}${exe}`
```

Comment:
update to tauri-apps url

Reply:
Yeah I need to move the repo first. I'm leaving it this way for now so it can be tested.

Reply:
Updated to use the new repo now :+1: